src/Apps/Artwork/Components/OtherWorks/ArtworkContexts/ArtworkGrids/ArtistArtworkGrid.tsx

```diff
@@ -24,10 +45,15 @@ export const ArtistArtworkGridFragmentContainer = createFragmentContainer<{
           artworks={artist.artworks_connection}
           columnCount={[2, 3, 4]}
           mediator={mediator}
+          onBrickClick={this.trackBrickClick.bind(this)}
         />
       </>
     )
-  }),
+  }
+}
+
+export const ArtistArtworkGridFragmentContainer = createFragmentContainer(
+  withContext(ArtistArtworkGrid),
```

Comment:
oo this is a nicer way than the more verbose + indentation with `<ContextConsumer...` style.

Reply:
Yah agree, tis was a nice experiment, but this is more flexible